src/widgets/refinement-list/refinement-list.tsx

```diff
@@ -161,6 +100,71 @@ type RefinementListSearchableCSSClasses = {
 export type RefinementListCSSClasses = RefinementListOwnCSSClasses &
   RefinementListSearchableCSSClasses;
 
+// Templates types
```

Comment:
```suggestion
```

If the grouping becomes too complex, we can move it into its own file, but I don't think that's needed for now